manipulation/src/functTest/java/org/jboss/gm/manipulation/SimpleProjectFunctionalTest.java

```diff
@@ -0,0 +1,58 @@
+package org.jboss.gm.manipulation;
+
+import static org.assertj.core.api.Assertions.assertThat;
+import static org.assertj.core.api.Assertions.tuple;
+
+import java.io.File;
+import java.io.FileReader;
+import java.io.IOException;
+import java.net.URISyntaxException;
+import java.nio.file.Path;
+
+import org.apache.maven.model.Model;
+import org.apache.maven.model.io.xpp3.MavenXpp3Reader;
+import org.codehaus.plexus.util.xml.pull.XmlPullParserException;
+import org.gradle.testkit.runner.BuildResult;
+import org.gradle.testkit.runner.GradleRunner;
+import org.gradle.testkit.runner.TaskOutcome;
+import org.junit.Rule;
+import org.junit.Test;
+import org.junit.rules.TemporaryFolder;
+
+public class SimpleProjectFunctionalTest {
+
+    @Rule
+    public TemporaryFolder tempDir = new TemporaryFolder();
+
+    @Test
+    public void ensureAlignmentFileCreated() throws IOException, URISyntaxException, XmlPullParserException {
```

Comment:
method name

Reply:
That's what I get for exercising  too much copy-pasta :laughing:

Reply:
Fixed